scripts/ci/docker-compose/integration-kerberos.yml

```diff
@@ -17,16 +17,55 @@
 ---
 version: "2.2"
 services:
-  kerberos:
-    image: apache/airflow:krb5-kdc-server-2020.08.28
-    hostname: kerberos
+  kdc-server-example-com:
+    build:
+      context: ..
+      dockerfile: ./dockerfiles/krb5-kdc-server/Dockerfile
+    image: krb5-kdc-server-example-com
```

Comment:
One thing I would like to change here - rather than building the kerberos image, we should publish it in airflow repo. This way we will have much faster tests.